dist/build.gradle

```diff
@@ -21,7 +21,6 @@ dependencies {
   compile project(':api')
   compile project(':core')
   compile project(':impl')
-  compile project(':management')
```

Comment:
Why not?  I thought the plan was to cut management out of the OSS distribution for GA once 107 didn't need it.